### PR TITLE
Fix spelling in documentation and add missing/invalid documentation.

### PR DESCRIPTION
**Pull request by armano2**
fixed merge #4062

### CODE REVIEW

File: framework/base/CStatePersister.php [hunk 1]
```diff
@@ -105,7 +105,8 @@ public function load()
 	 * Loads content from file using a shared lock to avoid data corruption when reading
 	 * the file while it is being written by save()
 	 *
-	 * @return string file contents
+	 * @param $filename
```
Comment (samdark) [inline framework/base/CStatePersister.php]: Should be `@param string $filename file contents`.

File: framework/cli/views/webapp/protected/models/LoginForm.php [hunk 2]
```diff
@@ -43,6 +43,8 @@ public function attributeLabels()
 	/**
 	 * Authenticates the password.
 	 * This is the 'authenticate' validator as declared in rules().
+	 * @param $attribute
```
Comment (samdark) [inline framework/cli/views/webapp/protected/models/LoginForm.php]: Need types and descriptions.

File: framework/gii/models/LoginForm.php [hunk 3]
```diff
@@ -19,6 +19,8 @@ public function rules()
 	/**
 	 * Authenticates the password.
 	 * This is the 'authenticate' validator as declared in rules().
+	 * @param $attribute
```
Comment (samdark) [inline framework/gii/models/LoginForm.php]: Need types and descriptions.

File: framework/web/filters/CHttpCacheFilter.php [hunk 4]
```diff
@@ -202,6 +202,7 @@ protected function sendCacheControlHeader()
 	/**
 	 * Generates a quoted string out of the seed
 	 * @param mixed $seed Seed for the ETag
+	 * @return string
```
Comment (samdark) [inline framework/web/filters/CHttpCacheFilter.php]: Need description.

File: framework/web/services/CWsdlGenerator.php [hunk 5]
```diff
@@ -361,6 +361,8 @@ protected function processMethod($method)
 
 	/**
 	 * @param string $type PHP variable type
+	 * @return mixed|string
```
Comment (samdark) [inline framework/web/services/CWsdlGenerator.php]: Need description.

File: framework/web/services/CWsdlGenerator.php [hunk 6]
```diff
@@ -441,9 +443,10 @@ protected function processType($type)
 	}
 
 	/**
-	* Parse attributes nillable, minOccurs, maxOccurs
-	* @param string $comment Extracted PHPDoc comment
-	*/
+	 * Parse attributes nillable, minOccurs, maxOccurs
+	 * @param string $comment Extracted PHPDoc comment
+	 * @return array
```
Comment (samdark) [inline framework/web/services/CWsdlGenerator.php]: Need description.

File: framework/web/services/CWsdlGenerator.php [hunk 7]
```diff
@@ -683,6 +687,7 @@ protected function addPortTypes($dom)
 	 * @param DOMDocument $dom Represents an entire HTML or XML document; serves as the root of the document tree
 	 * @param string $name method name
 	 * @param string $doc doc
+	 * @return mixed
```
Comment (samdark) [inline framework/web/services/CWsdlGenerator.php]: Need description.

File: framework/web/services/CWsdlGenerator.php [hunk 8]
```diff
@@ -725,6 +730,7 @@ protected function addBindings($dom)
 	 * @param DOMDocument $dom Represents an entire HTML or XML document; serves as the root of the document tree
 	 * @param string $name method name
 	 * @param array $headers array like array('input'=>array(MESSAGE,PART),'output=>array(MESSAGE,PART))
+	 * @return mixed
```
Comment (samdark) [inline framework/web/services/CWsdlGenerator.php]: Need description.

**Comment by armano2** [comment]
@samdark thanks for pointing out my mistakes :dancer:

**Comment by samdark** [comment]
Thanks!

**Pull request merged**